### PR TITLE
fix(escape character) Remove escape character if used into quote ``

### PR DESCRIPTION
**Pull request by julienmege**
closes(BS-17994)[https://bonitasoft.atlassian.net/browse/BS-17994]

### CODE REVIEW

**Comment by NathalieC** [comment]
retest this please

**Comment by tnguyen1** [comment]
retest this please

**Comment by NathalieC** [comment]
Some "\" remain in bpm-api.md.
Could you remove them?

File: md/bpm-api.md [hunk 1]
```diff
@@ -2489,19 +2492,19 @@ Use a GET method with filters and search terms to search for actorMembers.
...
+  * `actor_id` For example, retrieve the actorMembers related to the specified actor\_id. http://localhost:8080/bonita/API/bpm/actorMember?p=0&c=10&f=actor\_id%3d1
 
   You can also filter also on: 
 
-  * `member\_type=user|role|group|roleAndGroup` retrieve only the actorMembers of type user. `/API/bpm/actorMember?p=0&c=10&f=actor\_id%3d1&f=member\_type%3duser`
-  * `user\_id=:userId}`: retrieve only the actorMembers related to the specified user\_id. `/API/bpm/actorMember?p=0&c=10&f=actor\_id%3d1&f=user\_id%3d101`
-  * `role\_id=:roleId`: retrieve only the actorMembers related to the specified role\_id. `/API/bpm/actorMember?p=0&c=10&f=actor\_id%3d1&f=role\_id%3d101`
-  * `group\_id=:groupId`: retrieve only the actorMembers related to the specified group\_id. `/API/bpm/actorMember?p=0&c=10&f=actor\_id%3d1&f=group\_id%3d101`
+  * `member_type=user|role|group|roleAndGroup` retrieve only the actorMembers of type user. `/API/bpm/actorMember?p=0&c=10&f=actor_id%3d1&f=member_type%3duser`
+  * `user_id=:userId}`: retrieve only the actorMembers related to the specified user\_id. `/API/bpm/actorMember?p=0&c=10&f=actor_id%3d1&f=user_id%3d101`
```
Comment (educhastenier) [inline md/bpm-api.md]: extra `}` character

**Review by educhastenier — CHANGES_REQUESTED** [review]
There are still places where ``` character is still visible in the final rendering, on pages http://doc.rd.lan:50510/bonita/7.3/identity-api
http://doc.rd.lan:50510/bonita/7.3/bpm-api

Please fix them

**Review by tbouffard — APPROVED** [review]
Latest commits from @julienmege seems fixing the remarks noticed by @educhastenier

**Pull request merged**